src/iiif/image_handling.py

```diff
@@ -185,13 +185,22 @@ def crop_image(content, source_file, region, content_type):
     :param content_type: The content type of the image
     :return: The cropped image data
     """
-    if source_file or region.lower() == "full":
+    if source_file:
         return content
 
-    desired_x, desired_y, desired_width, desired_height = parse_region_string(region)
-    
     img = Image.open(BytesIO(content))
 
+    match region.lower():
+        case 'full':
+            return content
```

Comment:
Nu wordt de image ingeladen in Pillow, zelfs als er niets wordt gedaan. Het is de vraag of dat met het oog op de performance handig is. Kan je anders eens lokaal een test doen hoeveel ms het eigenlijk duurt om een grote file in te laden met een `Image.open(BytesIO())`. Als dat ook met grote files eigenlijk niets uitmaakt, dan is het prima zo. Als het wel een significant iets is, laten we dan die regio.lower() naar boven verhuizen.

Reply:
The `Image.open(BytesIO(content))` should be lazy loading the image data. So it will only load the image when needed. Quick and Dirty performance test showed no significant difference between moving the return content before the Image.open line. Both process 1.000.000 4mb images on my laptop in under 100 seconds with only half a second difference between the two methods.